spectator-ext-ipc/src/main/java/com/netflix/spectator/ipc/IpcTagKey.java

```diff
@@ -144,7 +166,11 @@
 
   /**
    * HTTP status code.
+   *
+   * @deprecated Use {@link #statusDetail} instead. This value will be removed in
```

Comment:
What was the reason for this? I'd rather keep this than start putting it into statusDetail.